plugins/module_utils/redfish_utils.py

```diff
@@ -1149,6 +1149,55 @@ def manage_power(self, command, resource_uri, action_name):
             return response
         return {'ret': True, 'changed': True}
 
+    def system_reset_to_defaults(self, command):
+        return self.reset_to_defaults(command, self.systems_uri,
```

Comment:
under systems instead we have `Systems/System.Embedded.1/Bios/Actions/Bios.ResetBios` ...

Reply:
I think it would be good to pull this out since Bios.ResetBios has different parameters. We did this for the Reset action on Manager and ComputerSystem because the same "ResetType" list applies to each.

Reply:
done